### ISSUE TITLE
Maintenance Testing for i18n of localization text

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/qa#1114]
Patches have been added to sims to fix https://github.com/phetsims/joist/issues/976. Can you please do a sanity check on each of these to make sure they are OK to deploy?

Release Candidate links

**area-model-algebra 1.3** (https://github.com/phetsims/joist/issues/976)

- [x] [area-model-algebra 1.3.4-rc.2](https://phet-dev.colorado.edu/html/area-model-algebra/1.3.4-rc.2/phet/area-model-algebra_all_phet.html)

**area-model-decimals 1.3** (https://github.com/phetsims/joist/issues/976)

- [x] [area-model-decimals 1.3.3-rc.1](https://phet-dev.colorado.edu/html/area-model-decimals/1.3.3-rc.1/phet/area-model-decimals_all_phet.html)

**area-model-introduction 1.3** (https://github.com/phetsims/joist/issues/976)

- [x] [area-model-introduction 1.3.3-rc.1](https://phet-dev.colorado.edu/html/area-model-introduction/1.3.3-rc.1/phet/area-model-introduction_all_phet.html)

**area-model-multiplication 1.3** (https://github.com/phetsims/joist/issues/976)

- [x] [area-model-multiplication 1.3.4-rc.1](https://phet-dev.colorado.edu/html/area-model-multiplication/1.3.4-rc.1/phet/area-model-multiplication_all_phet.html)

**arithmetic 1.2** (https://github.com/phetsims/joist/issues/976)

- [x] [arithmetic 1.2.3-rc.1](https://phet-dev.colorado.edu/html/arithmetic/1.2.3-rc.1/phet/arithmetic_all_phet.html)

**balancing-act 1.3** (https://github.com/phetsims/joist/issues/976)

- [x] [balancing-act 1.3.2-rc.1](https://phet-dev.colorado.edu/html/balancing-act/1.3.2-rc.1/phet/balancing-act_all_phet.html)

**graphing-lines 1.4** (https://github.com/phetsims/joist/issues/976)

- [x] [graphing-lines 1.4.3-rc.1](https://phet-dev.colorado.edu/html/graphing-lines/1.4.3-rc.1/phet/graphing-lines_all_phet.html)

**graphing-slope-intercept 1.2** (https://github.com/phetsims/joist/issues/976)

- [x] [graphing-slope-intercept 1.2.3-rc.1](https://phet-dev.colorado.edu/html/graphing-slope-intercept/1.2.3-rc.1/phet/graphing-slope-intercept_all_phet.html)

**greenhouse-effect 1.3** (https://github.com/phetsims/joist/issues/976)

- [x] [greenhouse-effect 1.3.3-rc.1](https://phet-dev.colorado.edu/html/greenhouse-effect/1.3.3-rc.1/phet/greenhouse-effect_all_phet.html)
- [x] [greenhouse-effect 1.3.3-rc.1 phet-io](https://phet-dev.colorado.edu/html/greenhouse-effect/1.3.3-rc.1/phet-io/greenhouse-effect_all_phet-io.html?phetioStandalone)
- [x] [greenhouse-effect 1.3.3-rc.1 phet-io Studio](https://phet-dev.colorado.edu/html/greenhouse-effect/1.3.3-rc.1/phet-io/wrappers/studio)

**hookes-law 1.1** (https://github.com/phetsims/joist/issues/976)

- [x] [hookes-law 1.1.1-rc.1](https://phet-dev.colorado.edu/html/hookes-law/1.1.1-rc.1/phet/hookes-law_all_phet.html)

**keplers-laws 1.2** (https://github.com/phetsims/joist/issues/976)

- [x] [keplers-laws 1.2.5-rc.1](https://phet-dev.colorado.edu/html/keplers-laws/1.2.5-rc.1/phet/keplers-laws_all_phet.html)
- [x] [keplers-laws 1.2.5-rc.1 phet-io](https://phet-dev.colorado.edu/html/keplers-laws/1.2.5-rc.1/phet-io/keplers-laws_all_phet-io.html?phetioStandalone)
- [x] [keplers-laws 1.2.5-rc.1 phet-io Studio](https://phet-dev.colorado.edu/html/keplers-laws/1.2.5-rc.1/phet-io/wrappers/studio)

**projectile-data-lab 1.0** (https://github.com/phetsims/joist/issues/976)

- [x] [projectile-data-lab 1.0.4-rc.1](https://phet-dev.colorado.edu/html/projectile-data-lab/1.0.4-rc.1/phet/projectile-data-lab_all_phet.html)
- [x] [projectile-data-lab 1.0.4-rc.1 phet-io](https://phet-dev.colorado.edu/html/projectile-data-lab/1.0.4-rc.1/phet-io/projectile-data-lab_all_phet-io.html?phetioStandalone)
- [x] [projectile-data-lab 1.0.4-rc.1 phet-io Studio](https://phet-dev.colorado.edu/html/projectile-data-lab/1.0.4-rc.1/phet-io/wrappers/studio)

**projectile-sampling-distributions 1.0** (https://github.com/phetsims/joist/issues/976)

- [x] [projectile-sampling-distributions 1.0.4-rc.1](https://phet-dev.colorado.edu/html/projectile-sampling-distributions/1.0.4-rc.1/phet/projectile-sampling-distributions_all_phet.html)
- [x] [projectile-sampling-distributions 1.0.4-rc.1 phet-io](https://phet-dev.colorado.edu/html/projectile-sampling-distributions/1.0.4-rc.1/phet-io/projectile-sampling-distributions_all_phet-io.html?phetioStandalone)
- [x] [projectile-sampling-distributions 1.0.4-rc.1 phet-io Studio](https://phet-dev.colorado.edu/html/projectile-sampling-distributions/1.0.4-rc.1/phet-io/wrappers/studio)

**Comment by jessegreenberg** [comment]
@KatieWoe pointed out that the bug is not reproducible in the published version of Arithmetic (1.2.2). I can't explain that, it would be good to understand why that is the case.

**Comment by jessegreenberg** [comment]
For the above note, we confirmed that the behavior is expected. The strings were previously hidden in the translataion utility, but were still to change with ?stringTest. So the test is no longer on hold.

I am not sure how to best verify that this is fixed, I will ask slack.

**Comment by KatieWoe** [comment]
Basic MR looks ok. As discussed with @jessegreenberg, there isn't a great way for QA to check that the actual issue is fixed. If something is found, let us know and we can retest for it. Otherwise, QA is done.

**Comment by jessegreenberg** [comment]
OK, thanks @KatieWoe!! I am going to go ahead with the deploys.  It sounds like there isn't a great way to verify this.

One thing I did to was verify that these strings are currently not available in the translation utility for sims that support dynamic locale. I will make sure that the strings are there after sims are deployed to production. Closing, and will continue in https://github.com/phetsims/joist/issues/976
